cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -373,6 +373,17 @@ function algoliaLocalizedProduct(parameters) {
                 this[attributeName] = aggregatedValueHandlers[attributeName](product, parameters);
             } else {
                 var config = algoliaProductConfig.attributeConfig_v2[attributeName];
+
+                // if there is no config for the attribute, but it's a custom or activeData attribute, we assume some defaults
+                if (!config && (attributeName.indexOf('custom.') > -1 || attributeName.indexOf('activeData.') > -1)) {
+                    // if the attribute is a custom attribute, we assume it's localized, you can override this behavior by adding a config for the attribute
+                    config = {
+                        attribute: attributeName,
+                        localized: true,
+                        variantAttribute: true
```

Comment:
Those properties are not used at that point, it's the jobs that use them:
- When `localized` is set to `false`, the jobs build a `baseModel` once, and then pass it in the `parameters` of this constructor. You can see above, if we find one of the attributes in this `baseModel`, it's used directly and to re-fetched. At this point in the code, the attribute will be fetched whatever the `localized` value (so it's what we want as the default).
- `variantAttribute` is used by the job to compute a list of `variantAttributes`, which is also passed in the `parameters` of this constructor. It's used by the `aggregatedValueHandlers.variants` handler, to know which attributes must go in the `variants`. The attribute will go at the record level anyway (I think that's ok as a default)

Reply:
Refactor by considering that..